### ISSUE TITLE
Druid should not require JVM time zone to be UTC

### ISSUE THREAD

**Issue by xvrl** [druid-io/druid#2619]
This issue is meant to aggregate all the problems related to JVM not running UTC:
- #228 Hadoop running non-utc
- #1204 Segment Granularities
- #1284 Integration Tests

**Comment by gianm** [comment]
Tranquility does work in non-UTC timezones, what it took to get there was,

- replacing all instances of `new DateTime(x)` with `new DateTime(x, ISOChronology.getInstanceUTC)`
- replacing all instances of `new Interval(long, long)` with `new Interval(long, long, ISOChronology.getInstanceUTC)`
- replacing all instances of `new Interval(string)` with `new Interval(string, ISOChronology.getInstanceUTC)`
- call `.withChronology(ISOChronology.getInstanceUTC)` on DateTimes received from "outside"
- call `.withChronology(ISOChronology.getInstanceUTC)` on Intervals received from "outside"

**Comment by gianm** [comment]
So essentially internally it's using UTC for everything and ignoring the `user.timezone` value.

**Comment by gianm** [comment]
re: "depending on whether you want to parse strings in UTC or in the user's timezone" IMO it would be better to parse strings in the user's timezone, so the latter form probably makes more sense.

**Comment by xvrl** [comment]
Agree with @gianm that we should always interpret strings in the user's timezone whenever possible (unless they already specify a time zone of course)

**Comment by binlijin** [comment]
:+1: , This is useful.

**Comment by leventov** [comment]
Closing this after #4611. If the problem persists let's open another issue
